zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/socket/Message.java

```diff
@@ -188,12 +190,28 @@
     INTERPRETER_INSTALL_RESULT,   // [s-c] Status of an interpreter installation
     COLLABORATIVE_MODE_STATUS,    // [s-c] collaborative mode status
     PATCH_PARAGRAPH,              // [c-s][s-c] patch editor text
+    SEQUENTIAL_RUN_STATUS,        // [s-c] sequential run status will be change
     NOTICE                        // [s-c] Notice
   }
 
+  // these messages will be ignored during the sequential run of the note
+  private static final List<OP> sequentialRunExcludeMessageList = Arrays.asList(
```

Comment:
We only doing search an element in a list. I think it's better to use HashSet here.
```suggestion
  private static final Set<OP> sequentialRunExcludeMessageList = new HashSet<>(Arrays.asList(
```

Reply:
done